src/Query/index.ts

```diff
@@ -108,22 +108,30 @@ export class Query {
     const parts: Part[] = [];
     const searchPropertyMapping = getPropertyMapping();
 
+    // DOP-3934 - test without having this hardcoded matches
```

Comment:
Is the plan to remove the commented code after some time of confirming things work as expected?

Reply:
see below response. lets leave this and remove as a separate test